backend/src/packages/mail/libs/maps/template-name-to-view.map.ts

```diff
@@ -1,11 +1,8 @@
-import { type IView } from '~/libs/packages/packages.js';
-
 import { TemplateName } from '../enums/enums.js';
-import { type TemplateNameValues } from '../types/types.js';
 import { plainView } from '../views/views.js';
 
-const templateNameToView: Record<TemplateNameValues, IView> = {
+const templateNameToView = {
```

Comment:
add a type for this map

Reply:
Why should there be a type for it, if it's inferred from the actual values?

Reply:
> add a type for this map
Why should there be a type for it, if it's inferred from the actual values?

Reply:
Added type, but I had to explicitly specify its keys and values to keep render parameter type.

Reply:
@mV22110702 Record<TemplateNameValues, IView> won't it be suitable?

Reply:
IView is generic, so, unfortunately, it's not going to work. I have tried to create generic type for the map that will infer the actual type, but no success at the moment.